tests/profiling_v2/test_gunicorn.py

```diff
@@ -30,7 +30,7 @@ def debug_print(*args):
 
 def _run_gunicorn(*args):
     cmd = (
-        ["ddtrace-run", "gunicorn", "--bind", "127.0.0.1:7643", "--chdir", os.path.dirname(__file__)]
+        ["ddtrace-run", "gunicorn", "--bind", "127.0.0.1:7644", "--chdir", os.path.dirname(__file__)]
```

Comment:
Do we get test failures because of port conflicts?

If so, is there any danger here of the gunicorn process not closing cleanly, leading the kernel TCP stack to leave the port bound until the timeout (~minutes)?

Reply:
I don't think we do, and it should return right away. But changed just in case.